ffi/custom_passes.cpp

```diff
@@ -673,6 +680,7 @@ struct RefPrunePass : public FunctionPass {
      */
     bool findFanoutDecrefCandidates(CallInst *incref,
                                     BasicBlock *cur_node,
+                                    SmallBBSet &bad_blocks,
```

Comment:
Add parameter description to docstring.

Reply:
fixed in https://github.com/numba/llvmlite/pull/644/commits/fa6092b8de364fe7a04c00a08523cc1c1e027783